src/include/security/levels.rb

```diff
@@ -115,7 +115,7 @@ def initialize_security_levels(include_target)
             "FAIL_DELAY"                                => "6",
             "GID_MAX"                                   => "60000",
             "GID_MIN"                                   => "1000",
-            "DISPLAYMANAGER_SHUTDOWN"                   => "root",
+            "AllowShutdown"                             => "Root",
```

Comment:
Yes. From /usr/share/kde4/config/kdm/kdmrc
```
# Who is allowed to shut down the system. This applies both to the
# greeter and to the command sockets.
# "None" - no "Shutdown..." menu entry is shown at all
# "Root" - the root password must be entered to shut down
# "All" - everybody can shut down the machine
# Default is All
```
Actually, I'm quite sure it's case insensitive in the end but according to documentation, it should be "Root".